src/index.ts

```diff
@@ -138,9 +193,14 @@ async function create(argv: yargs.Arguments<any>) {
         name: basename,
         package: basename.toLowerCase(),
       },
+      extraOptions,
     } as Options,
   );
 
+  if (options.modules.navigation) {
+    options.modules.gestureHandler = true;
```

Comment:
+ `react-native-screens` when `nativeStack` is chosen

```suggestion
    options.modules.gestureHandler = true;
    options.modules.reanimated = true;
```